Build.ps1

```diff
@@ -0,0 +1,129 @@
+Param(
+    # Archive containing the Ceph Windows binaries, will be fetched using scp.
+    # Can be a local path, a UNC path or a remote scp path.
+    [Parameter(Mandatory=$true)]
+    [string]$CephZipPath,
+
+    # The thumbprint of the X509 certificate used for signing the WNBD driver
+    # and the MSI installer
+    [string]$SignX509Thumbprint = $null
+)
+
+$ErrorActionPreference = "Stop"
+
+function SetVCVars($version="2019", $platform="x86_amd64") {
+    pushd "$ENV:ProgramFiles (x86)\Microsoft Visual Studio\$version\Community\VC\Auxiliary\Build"
+    try {
+        cmd /c "vcvarsall.bat $platform & set" |
+        foreach {
+          if ($_ -match "=") {
+            $v = $_.split("="); set-item -force -path "ENV:\$($v[0])"  -value "$($v[1])"
+          }
+        }
+    }
+    finally {
+        popd
+    }
+}
+
+function Expand7z($archive, $outputDir = ".") {
+    pushd .
+    try {
+        cd $outputDir
+        &7z.exe x -y $archive

```

Comment:
Having 7zip as an additional dependency might be undesired. Still, we talked a bit about it and apparently it's much faster than Expand-Archive. We can do a follow up that uses Expand-Archive as a fallback.

Reply:
By comparison, Expand-Archive takes 13s on my env vs 3s of 7z. We're not going to spare a dependency if MSFT did a crappy job. Said that, testing for 7z and falling back to Expand-Archive is something easy to add (done)

Reply:
Thanks to this petrutlucian94/ceph#8 by eliminating all the extra zip content, unzip time difference went down to 2s, not worth keeping the extra dependency, so Expand-Archive it is